Source/Libraries/GSF.PhasorProtocols/UI/WPF/ViewModels/InputWizardDevices.cs

```diff
@@ -56,2021 +56,2063 @@
+            // If an error occurred during validation,
+            // set the PdcMessage to that error message as well
+            if (!string.IsNullOrEmpty(errorMessage))
+                PdcMessage = errorMessage;
         }
 
-        /// <summary>
-        /// Generates connection string to save into database by merging connection string and alternate command channel.
-        /// </summary>
-        /// <returns>string, to store into database.</returns>
-        private string GenerateConnectionString()
-        {
-            string connectionString = ConnectionString;
-
-            if (string.IsNullOrEmpty(AlternateCommandChannel))
-                return connectionString;
+        // Update the error message
+        m_errorMessages["PdcAcronym"] = errorMessage;
 
-            if (!connectionString.EndsWith(";"))
-                connectionString += ";";
+        // Notify of changes in validation state
+        OnPropertyChanged("PdcAcronym");
+    }
 
-            connectionString += $"commandchannel={{{AlternateCommandChannel}}}";
+    /// <summary>
+    /// Generates connection string to save into database by merging connection string and alternate command channel.
+    /// </summary>
+    /// <returns>string, to store into database.</returns>
+    private string GenerateConnectionString()
+    {
+        string connectionString = ConnectionString;
 
+        if (string.IsNullOrEmpty(AlternateCommandChannel))
             return connectionString;
-        }
-
-        private void DisplayPopup(string message, string source, MessageBoxImage image)
-        {
-            if (m_dispatcher.Thread != Thread.CurrentThread)
-                m_dispatcher.BeginInvoke((Action<string, string, MessageBoxImage>)DisplayPopup, message, source, image);
-            else
-                Popup(message, source, image);
-        }
 
-        #endregion
+        if (!connectionString.EndsWith(";"))
+            connectionString += ";";
 
-        #region [ Static ]
+        connectionString += $"commandchannel={{{AlternateCommandChannel}}}";
 
-        // Fields
-        private static ManualResetEvent s_responseWaitHandle;
-        private static readonly string[] s_commonVoltageLevels = { "44", "69", "115", "138", "161", "169", "230", "345", "500", "765", "1100" };
+        return connectionString;
+    }
 
-        #endregion
+    private void DisplayPopup(string message, string source, MessageBoxImage image)
+    {
+        if (m_dispatcher.Thread != Thread.CurrentThread)
+            m_dispatcher.BeginInvoke((Action<string, string, MessageBoxImage>)DisplayPopup, message, source, image);
+        else
+            Popup(message, source, image);
     }
-}
+
+    #endregion
+
+    #region [ Static ]
+
+    // Fields
+    private static ManualResetEvent s_responseWaitHandle;
+    private static readonly string[] s_commonVoltageLevels = ["44", "69", "115", "138", "161", "169", "230", "345", "500", "765", "1100"];
```

Comment:
Same issue, different spot.

```suggestion
    private static readonly string[] s_commonVoltageLevels = ["1100", "765", "500", "345", "230", "169", "161", "138", "115", "69", "44"];
```